controllers/grid/users/reviewer/ReviewerGridRow.php

```diff
@@ -215,6 +216,66 @@ public function initialize($request, $template = null)
                     )
                 );
             }
+            if ($reviewAssignment->getDateConfirmed() == null) {
+                $context = $request->getContext();
+                $vueModalArgs = [
+                    'description' => $submission->getCurrentPublication()->getLocalizedTitle(null, 'html'),
+                    'logResponseForm' => [
+                        'id' => 'logResponseForm',
+                        'method' => 'POST',
+                        'action' => $router->url($request, null, null, 'addLog', null, $actionArgs),
+                        'fields' => [
```

Comment:
Is not this opportunity to create rest API endpoint - instead of using these grid legacy endpoints? 

Basically everytime we replace something with Vue.js which needs to interact with backend - we also create rest api endpoint, which is generally useful for interacting with OJS.